### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Guillaume Fragnière and Killian Viquerat

### DISCUSSION THREAD

**Discussion by GFragniere** [heig-vd-dai-course/heig-vd-dai-course#458] [Show and tell]
For this second project, we wanted to recreate the Tron game with UDP, with a maximum of 4 players, each having their own color.
You will be able to select your color within the possible colors and chose your name, and compete against your opponents.
We will also be using the Lanteran library, a TUI (Terminal User Interface) to allow the use of a semi-graphical intefrace.

The Github repository of the project: https://github.com/GFragniere/Practical_work_2_Tron

**Comment by GeraudSilvestri** [comment]
It's good for me.

Remember to start simple and once that's up and working to add more complex features to it.

**Comment by GFragniere** [comment]
Link to repository: https://github.com/GFragniere/Practical_work_2_Tron
hash of last commit: 9d93cd7902764073755685b00f0bbf7c3d4548a6

Note that we did not use the Lanterna library as mentionned in the first message, but we will discuss that during the presentation.
